int16 my_classifier
---
bool success
